frontend/public/style/_common.scss

```diff
@@ -378,6 +378,7 @@
 .co-select-to-copy {
   cursor: copy;
   user-select: all;
+  -moz-user-select: all;
```

Comment:
Also add `-webkit-user-select: all;` since Safari requires the prefix.  
note: `-ms-user-select: all` does not seem to work on Edge.

Reply:
Good catch. Thanks, updated